spec_helper.rb

```diff
@@ -30,6 +30,10 @@ module SpecHelper
 my_main = self
 RSpec.configure do |config|
   config.before(:each) do |current_test|
+    if current_test.id =~ %r{(fastlane\/spec\/(?!lane_manager))|(scan\/spec\/test_command_generator)}
+      allow(FastlaneCore::FastlaneFolder).to receive(:path).and_return(nil)
+    end
```

Comment:
@mfurtak @KrauseFx Thank you for your feedback.
I'll put it to each test that needs the mock 👍